tt_eager/tt_lib/csrc/operations/primary/module.hpp

```diff
@@ -754,9 +754,11 @@ void py_module(py::module& m_primary) {
         py::arg("gamma").noconvert() = std::nullopt,
         py::arg("beta").noconvert() = std::nullopt,
         py::kw_only(),
+        py::arg("output").noconvert() = std::nullopt,
         py::arg("mean").noconvert() = std::nullopt,
         py::arg("rstd").noconvert() = std::nullopt,
         py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
```

Comment:
same as here

https://github.com/tenstorrent/tt-metal/pull/9804#discussion_r1661859299

Reply:
```C++
tt_eager/tt_lib/csrc/operations/primary/module.hpp

py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
->
py::arg("memory_config").noconvert() = std::nullopt,

```
https://github.com/tenstorrent/tt-metal/pull/9636/commits/b2168abdbefbb42af71a7c44cb5cbae4310e2bd3#diff-635a2960d38f45b47d93f3be3e4cc45b69c32e0e904f69d20486a47ee7803b79

I have modified it to receive `memory_config` like other implementations: `const std::optional<MemoryConfig>& memory_config = std::nullopt`.